packages/block-library/src/list/editor.scss

```diff
@@ -1,5 +1,5 @@
-.editor-styles-wrapper .block-library-list ul,
-.editor-styles-wrapper .block-library-list ol {
+.editor-styles-wrapper div[data-type="core/list"] ul,
+.editor-styles-wrapper div[data-type="core/list"] ol {
```

Comment:
Ideally we should remove these styles... They make it harder for themes to style the list block in the editor. Anything stopping us from removing it?

Reply:
Agree, we should remove them. However what's stopping us is, once again, the dreaded CSS bleed. Bot ul, ol and li are separately styled in an unscoped fashion by wp-admin itself. 

The two best fixes are to a) use Shadow DOM, and b) scope the wp-admin styles. We should do both long term.

Near term all we can do is reduce the specificity of these. 

It is worth noting that these are in `editor.scss`, so at least they won't affect the theme.

Reply:
@jasmussen I still don't get why the admin css matters if the themes provides something like

```css
.editor-styles-wrapper ul,
.editor-styles-wrapper ol {
  margin: ...;
  padding: ...;
}
```

Currently `master` will override the theme styles? Can't I just remove the extra specificity here so it properly acts as default styling.